youtube-channels.csv

```diff
@@ -546,3 +546,37 @@ UCb4XH3v5SKMQXJWL_lAiUbg, beatsolow, https://www.youtube.com/channel/UCb4XH3v5SK
 UCxT-uvIzsMlvGXDQu0omuhQ, Fern, https://www.youtube.com/channel/UCxT-uvIzsMlvGXDQu0omuhQ
 UCtTly50qDGxwwcNw3u0sx3Q, Dito Korkelia, https://www.youtube.com/channel/UCtTly50qDGxwwcNw3u0sx3Q
 UC-tzG7z4619UJIANeyYWXDg, Pasti's Library, https://www.youtube.com/channel/UC-tzG7z4619UJIANeyYWXDg
+EuphoricHardStyleZ, EuphoricHardStyleZ , https://www.youtube.com/user/EuphoricHardStyleZ
+UCBuN7YIHjB40uXezntguDfQ, ConnecteD {Hard Dance Music}, https://www.youtube.com/channel/UCBuN7YIHjB40uXezntguDfQ
+TrancesMaster, TrancesMaster, https://www.youtube.com/user/TrancesMaster
+someuncertainsir, some uncertain sir, https://www.youtube.com/c/someuncertainsir
+SkankandbassUK, Skankandbass, https://www.youtube.com/c/SkankandbassUK
+AnalogueNetwork, Analogue Network, https://www.youtube.com/c/AnalogueNetwork
+HeavenlyHouse, HeavenlyHouse, https://www.youtube.com/user/HeavenlyHouse
+DistrictBass, DistrictBass, https://www.youtube.com/c/DistrictBass
+UKF, UKF, https://www.youtube.com/c/UKF
+dnbtv, Drum&BassArena, https://www.youtube.com/dnbtv
+UCL8Fd-ZYgPBUG8j8VGLEtLA, Zuka Zhvania , https://www.youtube.com/channel/UCL8Fd-ZYgPBUG8j8VGLEtLA
+EnchantedRhythms, EnchantedRhythms, https://www.youtube.com/c/EnchantedRhythms
+@hardvision, HARD VISION, https://www.youtube.com/@hardvision
+TeklaDuduchava, Tekla Duduchava, https://www.youtube.com/c/TeklaDuduchava
+LocalRequest4991, Local Request 4991, https://www.youtube.com/c/LocalRequest4991
+InspectorDubplate, Inspector Dubplate, https://www.youtube.com/user/InspectorDubplate
+Housechild, Housechild, https://www.youtube.com/c/Housechild
+Pretiosum, Pretiosum, https://www.youtube.com/c/Pretiosum
+SuicideSheeep, SuicideSheeep, https://www.youtube.com/user/SuicideSheeep
+funkypanda, funkypanda, https://www.youtube.com/c/funkypanda
+@TastyNetwork, Tasty, https://www.youtube.com/@TastyNetwork
+RokTorkar, Rok Torkar, https://www.youtube.com/c/RokTorkar
+TInTechno , T In Techno, https://www.youtube.com/c/TInTechno
+UC0DuVZILjFGM-Iln6wp9DQQ, HardTune Uploaderz [HTuZ], https://www.youtube.com/channel/UC0DuVZILjFGM-Iln6wp9DQQ
+FrenchcoreHardcore, Frenchcore Hardcore, https://www.youtube.com/c/FrenchcoreHardcore
+@ourgrnd, OURGRND, https://www.youtube.com/@ourgrnd
+TrueHardtranceSounds, TrueHardtranceSounds, https://www.youtube.com/user/TrueHardtranceSounds
+TrueHardstyleSounds, TrueHardstyleSounds, https://www.youtube.com/user/TrueHardstyleSounds
+TheLegendaryUploadzz, TheLegendaryUploadzz, https://www.youtube.com/c/TheLegendaryUploadzz
+TheHardstyleMusicz, TheHardstyleMusicz, https://www.youtube.com/c/TheHardstyleMusicz
+Davide1437, HOUSE ALL TIMES, https://www.youtube.com/user/Davide1437
+nareku, Nareku, https://www.youtube.com/c/nareku
+UCmz8lDqD2RvhSirXNsz_raw, UNSR & DEMENTORS UNITY, https://www.youtube.com/channel/UCmz8lDqD2RvhSirXNsz_raw
+RawhausTrax, RawhausTrax / David Mast, https://www.youtube.com/user/RawhausTrax
```

Comment:
rows need to start with the channel ID
also I see there are a lot of duplicates

Reply:
how do i get the channel id? Cause youtube doesn't show them anymore

Reply:
use https://commentpicker.com/youtube-channel-id.php

Reply:
thanks